pkg/apis/app/v1alpha1/types.go

```diff
@@ -20,7 +20,10 @@ Collection of common types among CRDs
 
 // Image is a definition of a Docker image
 type Image struct {
-	Domain    string `json:"domain,omitempty"`
+	Domain string `json:"domain,omitempty"`
```

Comment:
The format is wrong here?

Reply:
The format change is affected by adding a comment below.